ffs/scheduler/scheduler.go

```diff
@@ -422,7 +428,29 @@ func (s *Scheduler) executeQueuedStorage(j ffs.StorageJob) {
 
 	// Execute
 	s.l.Log(ctx, "Executing job %s...", j.ID)
-	info, dealErrors, err := s.executeStorage(ctx, a, j)
+	dealUpdates := make(chan ffs.FilStorage)
```

Comment:
Maybe make it buffered to leave some space because on start a lot of things might get pushed fast? e.g: `make(chan ffs.FilStorage, 1000)`?